### PR TITLE
Set up Prettier to run in ESLint

### PR DESCRIPTION
**Pull request by alexlafroscia**
Related to #10

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -4,10 +4,20 @@ module.exports = {
...
+    'prettier'
+  ],
+  plugins: [
+    'prettier'
+  ],
   env: {
     browser: true
   },
   rules: {
+    'prettier/prettier': ['error', {
```
Comment (alexlafroscia) [inline .eslintrc.js:18]: I kept the same 120-width setting that we agreed on earlier, and also set it up to prefer single quotes, which I think it a better style for JS code. My preference is single quotes for JS and within handlebars and double quotes within HTML.

**Comment by Alonski** [comment]
Ok no problem. I might do the same for my company project as well. Do I need to run the linter myself before committing or is it setup to do that automatically?

**Comment by alexlafroscia** [comment]
> Do I need to run the linter myself before committing or is it setup to do that automatically?

It should run automatically as part of `ember test` since we have `ember-cli-eslint` installed already.  You can add a git hook to run the tests before committing if that's your thing, but either way Travis should catch any issues.

**Comment by alexlafroscia** [comment]
Rebased to fix merge conflict with the test-fix PR.

**Pull request merged**